File: Programmers/3진법 뒤집기.py
def solution(n):
    answer = 0
    temp = ""
    while n > 0:
        remain = n % 3
        n = n // 3
        temp += str(remain)

    for i in range(len(temp)):
        answer += (3 ** (len(temp) - 1 - i)) * int(temp[i])

    return answer

File: Programmers JS/3진법 뒤집기.js
function solution(n) {
  var answer = 0;
  let third = [];
  while (n > 0) {
    let add = n % 3;
    n = parseInt(n / 3);
    third.push(add);
  }

  third.reverse().forEach((el, idx) => {
    answer += el * 3 ** idx;
  });

  return answer;
}
